crates/ruma-events/tests/it/message.rs

```diff
@@ -3,21 +3,18 @@
 use assert_matches2::assert_matches;
 use assign::assign;
 use js_int::uint;
-#[cfg(feature = "unstable-msc3954")]
-use ruma_common::events::emote::EmoteEventContent;
-use ruma_common::{
-    events::{
-        message::{MessageEventContent, TextContentBlock, TextRepresentation},
-        relation::InReplyTo,
-        room::message::Relation,
-        AnyMessageLikeEvent, MessageLikeEvent,
-    },
-    owned_event_id,
-    serde::CanBeEmpty,
-    MilliSecondsSinceUnixEpoch,
+use ruma_common::{owned_event_id, serde::CanBeEmpty, MilliSecondsSinceUnixEpoch};
+use ruma_events::{
+    message::{MessageEventContent, TextContentBlock, TextRepresentation},
+    relation::InReplyTo,
+    room::message::Relation,
+    AnyMessageLikeEvent, MessageLikeEvent,
 };
 use serde_json::{from_value as from_json_value, json, to_value as to_json_value};
 
+#[cfg(feature = "unstable-msc3954")]
+use crate::emote::EmoteEventContent;
```

Comment:
Shouldn't an integration test use `use ruma_events::emote::EmoteEventContent;`?

It's the same for most integration tests of this crate.